akka-typed-testkit/src/main/scala/akka/typed/testkit/Effects.scala

```diff
@@ -92,7 +92,7 @@ class EffectfulActorContext[T](_name: String, _initialBehavior: Behavior[T], _ma
     effectQueue.offer(Spawned(name))
     super.spawn(behavior, name)
   }
-  override def stop(child: ActorRef[_]): Boolean = {
+  override def stop[U](child: ActorRef[U]): Boolean = {
```

Comment:
good 👍 
Is this class also touched by java people? that style API is better for java AFAIR